.circleci/config.yml

```diff
@@ -1,79 +1,137 @@
 version: 2.1
 
 executors:
-  docker-node:
-    parameters:
-      version:
-        default: "lts"
-        type: string
+  linux: &linux-executor
     docker:
-      - image: cimg/node:<<parameters.version>>
+      - image: cimg/base:2020.01
+  macos: &macos-executor
+    macos:
+      xcode: 11.3.0
+  windows: win/default
+
+
+orbs:
+  win: circleci/windows@2.4.0
+  node: circleci/node@4.5.1
 
 commands:
   npmrc:
     steps:
       - run:
           name: Create .npmrc
           command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" >> .npmrc
-  install:
+
+  setup_git:
     steps:
       - run:
-          name: Install
-          command: npm install
-  test:
+          name: "Setup git environment"
+          command: git config --global core.autocrlf false
+
+  install_deps:
+    description: Install dependencies
     steps:
       - run:
-          name: Test
-          command: npm test
-  release:
+          name: Install dependencies
+          command: npm install
+
+  install_node:
+    description: Install nodejs
+    parameters:
+      os:
+        type: executor
+      node_version:
+        type: string
     steps:
-      - run:
-          name: Release
-          command: npx semantic-release
+      - when:
+          condition:
+            or:
+              - equal: [ *macos-executor, << parameters.os >> ]
+              - equal: [ *linux-executor, << parameters.os >> ]
+          steps:
+            - node/install:
+                node-version: << parameters.node_version >>
+
+      - unless:
```

Comment:
@anthogez do you know if we are able to write the same expression without using unless and the same condition one more time. It would be nice if they had something similar to if-else-statements.